test/FunctionalTests/Microsoft.OData.Edm.Tests/Csdl/CsdlReaderTests.cs

```diff
@@ -2161,6 +2166,85 @@ public void ImportSchemaElementsOnlyFromReferencedNamespacesl()
             Assert.Null(personType);
         }
 
+        // We currently do not support reading the scale and srid values if the value is variable or Variable 

```

Comment:
Is this statement true given the test named `ReadAnnotationWithoutSpecifiedValueAndUsePrimitiveDefaultValues`?

Reply:
Doesn't that constitute reading the values?